frontend/src/libs/components/chat-sidebar/chat-sidebar.tsx

```diff
@@ -0,0 +1,51 @@
+import { IconName } from '#libs/enums/enums.js';
+import { useCallback, useSearch, useState } from '#libs/hooks/hooks.js';
+
+import { Icon, Search } from '../components.js';
+import { ChatElement } from './chat-element/chat-element.js';
+import styles from './styles.module.scss';
+
+//mocked data
+const chats = [
+  { id: 1, name: 'Relationship' },
+  { id: 2, name: 'Friends' },
+  { id: 3, name: 'Family' },
+  { id: 4, name: 'Work' },
+];
+
+const ChatSidebar: React.FC = () => {
+  const [filter, setFilter] = useState('');
+  const handleFilterChange = useCallback(
+    (event: React.ChangeEvent<HTMLInputElement>): void => {
+      setFilter(event.target.value);
+    },
+    [],
+  );
+  const filteredChats = useSearch(filter, chats, 'name');
```

Comment:
Filter should be inside hook and handle with useAppForm

Reply:
@v0ldemar01  so how it should be? As i understand i don`t need this filteredChats and what about useAppForm: i should wrap chat-sidebar component with form?

Reply:
done